### ISSUE TITLE
Maximum number of retries exceeded

### ISSUE THREAD

**Issue by Amirkaaa** [airtai/faststream-gen#128]
I ran next code as was given in example.
```
faststream_gen "Create a FastStream application using localhost broker for testing and use the default port number. 
It should consume messages from the 'input_data' topic, where each message is a JSON 
encoded object containing a single attribute: 'data'. While consuming from the topic,
increment the value of the data attribute by 1. Finally, send message to the 'output_data' topic."
```

Then got this result. 
```
✘ Error: Maximum number of retries (3) exceeded. Unable to fix the following issues. Please try again...                                                                 
                                                                                                                                                                         
                                                                                                                                                                         
Please check if your application description is missing some crutial information:                                                                                        
 - Description of the messages which will be produced/consumed                                                                                                           
 - At least one topic                                                                                                                                                    
 - The business logic to implement while consuming/producing the messages                                                                                                
                                                                                                                                                                         
                                                                                                                                                                         
If you're unsure about how to construct the app description, consider the following example for guidance                                                                 
                                                                                                                                                                         
APPLICATION DESCRIPTION EXAMPLE:                                                                                                                                         
Create a FastStream application using localhost broker for testing and use the default port number.                                                                      
It should consume messages from the 'input_data' topic, where each message is a JSON encoded object containing a single attribute: 'data'.                               
For each consumed message, create a new message object and increment the value of the data attribute by 1. Finally, send the modified message to the 'output_data' topic.
                                                                                                                                                                         
                                                                                                                                                                         
                                                                                                                                                                         
 Tokens used: 49869                                                                                                                                                      
 Total Cost (USD): $0.15187
 ```
Project generation not working, I waste my money to get this error. I just wanna see example what I will get, can you share link to github repository with a simple result of generation?

**Comment by harishmohanraj** [comment]
Hi @Amirkaaa,

Looks like the newer version of pydantic which was released a few hours ago is causing some incompatibility issues with our main FastStream library. Our team is working on it. 

In the meantime, you can try generating the project by uninstalling the latest pydantic version (2.4.0) and installing the previous version (2.3.0).

If you are still having problems, please do not hesitate to contact me. I'm more than willing to assist!

**Comment by harishmohanraj** [comment]
Hi @Amirkaaa ,

The issue has been resolved in the latest faststream-gen version (0.1.2). Please update to the latest version, and let me if you encounter any further problems.

**Comment by Amirkaaa** [comment]
I've tried a lot of combinations like, 
- pydantic@2.4.0 with _gen@1.3
- pydantic @2.3.0 with _gen@1.3
- pydantic@2.3.0 with _gen@1.2. and so on.
I've decided to abandon this idea and just created my custom template. Sorry but I don't have time to debug.
